openapi/platform/platform.yaml

```diff
@@ -1660,6 +1886,384 @@ paths:
+                        - type: businessCategories
+                          id: 'active:diving:freediving'
+                    type: salesAccounts
+  '/salesAccounts/{id}/relationships/customData':
+    parameters:
+      - schema:
+          type: string
+        name: id
+        in: path
+        required: true
+    get:
+      summary: Get Sales Account Custom Data ID
+      responses:
+        '200':
+          description: OK
+          content:
+            application/vnd.api+json:
+              schema:
+                type: object
+                properties:
+                  links:
+                    type: object
+                    properties:
+                      related:
+                        type: string
+                      self:
+                        type: string
+                  data:
+                    type: object
+                    properties:
+                      type:
+                        type: string
+                        example: salesAccountCustomData
+                      id:
+                        type: string
+                        example: AG-1234123412
+              examples:
+                Relationship Example:
+                  value:
+                    links:
+                      related: 'http://example.com/platform/salesAccounts/AG-1234123412/customDatas'
+                      self: 'http://example.com/platform/salesAccounts/AG-1234567812/relationships/customDatas'
+                    data:
+                      type: salesAccountCustomData
+                      id: AG-1234123412
+      operationId: get-salesAccountCustomDataID-by-salesAccount
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Proposed`
+
+        Returns custom metadata about the sales account.
+
+        Attributes can be set up at the partner administration page in Partner Center.
+      security:
+        - OAuth2:
+            - sales.account
+      x-lifecycle:
+        status: proposed
+      tags:
+        - Sales Account Custom Data
+      parameters:
+        - schema:
+            type: string
+            example: Bearer <Access Token>
+            pattern: ^Bearer\s\S+
+          in: header
+          name: Authorization
+          description: A Bearer access token to identify the user the app is acting on behalf of. See the Authorization guide for details.
+          required: true
+    options:
+      summary: ''
+      operationId: options-salesAccountCustomDataID-by-salesAccount
+      responses:
+        '204':
+          description: No Content
+      description: 'Used solely for [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) the OPTIONS request returns the list of possible HTTP methods and other headers that browsers use to protect user''s security. You should not call this operation directly. '
+      tags:
+        - Sales Account Custom Data
+  '/salesAccounts/{id}/customData':
```

Comment:
How should we describe the difference between `/salesAccounts/{id}/customData` and `/salesAccountCustomData/{id}`?
Should we allow updates to this path as well so they are the exact same?

Reply:
I think we discussed the update on `/salesAccounts/{id}/customData`. We would have to add support for that in the api2go, the problem is that we 'd have to add support for a lot of types of relationship updates. Thats the reason we left salesAccountCustomData as a top-level entity.